src/app/shared/components/display-table/metadata-table.component.ts

```diff
@@ -16,7 +16,7 @@ import { ReportUtil } from '../../util/report-util';
 export class MetadataTableComponent implements OnChanges {
   protected readonly ReportUtil = ReportUtil;
 
-  @Input({ required: true }) report!: Report | Checkpoint;
+  @Input({ required: true }) report?: Report | Checkpoint;
```

Comment:
If it's required, then it can't be undefined

Reply:
Added the change

Reply:
Yes it should even